app/components/pages/SubmissionWizard/steps/Editors/EditorsContent.js

```diff
@@ -41,6 +42,12 @@ const ValidationMessage = ({ message }) => (
 )
 
 class EditorsContent extends React.Component {
+  constructor() {
+    super()
+    this.START_GROW_INDEX = 2
+    this.MAX_REVIEWERS = 6
```

Comment:
Is it necessary for these to be hard coded variables within the constructor?

The concept of _when_ a new line gets rendered could belong to a particular method (e.g. `handleSuggestedReviewersChanged`) & the restriction of "up to 6" could live in the render method?

I don't think `START_GROW_INDEX` is being used anywhere?

Reply:
It's usually not a good idea to declare constants where you are going to use them. They are defined in a constructor or a dedicated place so you can easily change them in the future without going through the code.

Reply:
Feel free to refactor!